junos/func_resource_interface_physical.go

```diff
@@ -0,0 +1,77 @@
+package junos
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+func setIntEsi(setPrefix string, esiParams []interface{},
```

Comment:
Add this function in resource_interface_physical.go file after setInterfacePhysical function